src/workerd/api/web-socket.c++

```diff
@@ -391,6 +391,10 @@ WebSocket::Accepted::Accepted(Hibernatable wsParam, Native& native, IoContext& c
 
 kj::Promise<void> WebSocket::Accepted::createAbortTask(Native& native, IoContext& context) {
   try {
+    // whenAborted() is theoretically not supposed to throw, but some code paths, like
```

Comment:
nit: At a glance, it feels a bit weird for the log in question to be almost 20 lines away. It makes sense of course, but it may be helpful to someone looking at this for the first time if we change the wording a bit "_-- but log in the catch if..._" so the focus doesn't go to the immediately surrounding lines.